pkg/remote/deploy_pfe.go

```diff
@@ -178,21 +195,57 @@ func setPFEEnvVars(codewind Codewind, deployOptions *DeployOptions) []corev1.Env
 	}
 }
 
-// setPFEVolumes returns the 3 volumes & corresponding volume mounts required by the PFE container:
+func createCodewindPVC(codewind Codewind, deployOptions *DeployOptions, storageClass string) corev1.PersistentVolumeClaim {
+
+	labels := map[string]string{
+		"app":               KeycloakPrefix,
+		"codewindWorkspace": codewind.WorkspaceID,
+	}
+
+	pvc := corev1.PersistentVolumeClaim{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "PersistentVolumeClaim",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   PFEPrefix + "-pvc-" + codewind.WorkspaceID,
```

Comment:
We're passing in `codewind.PVCName` as `PVC_NAME` in the PFE Deployment, rather than `PFEPrefix + "-pvc-" + codewind.WorkspaceID`

Reply:
change var to point to `PFEPrefix + "-pvc-" + codewind.WorkspaceID`